dockerfile/Dockerfile

```diff
@@ -13,41 +13,60 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License
-
+#
 # ------------------------------------------------------------------------
 
-FROM wso2/wso2base:latest
-MAINTAINER dev@wso2.org
-
+FROM ubuntu:16.04
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
 ENV DEBIAN_FRONTEND noninteractive
 
-# Build time arguments
-ARG WSO2_SERVER
-ARG WSO2_SERVER_VERSION
-ARG WSO2_SERVER_PROFILE
-ARG WSO2_ENVIRONMENT
-ARG WSO2_DEPLOYMENT_PATTERN
-ARG HTTP_PACK_SERVER
-ARG PLATFORM
-
-# Carbon server information
-ENV WSO2_SERVER=${WSO2_SERVER:-wso2is} \
-    WSO2_SERVER_VERSION=${WSO2_SERVER_VERSION:-5.3.0} \
-    WSO2_SERVER_PROFILE=${WSO2_SERVER_PROFILE:-default} \
-    WSO2_ENVIRONMENT=${WSO2_ENVIRONMENT:-dev} \
-    WSO2_DEPLOYMENT_PATTERN=${WSO2_DEPLOYMENT_PATTERN:-1} \	
-    HTTP_PACK_SERVER=${HTTP_PACK_SERVER} \
-    PLATFORM=${PLATFORM}
-
-COPY scripts/*.sh /usr/local/bin/
-
-# Execute configuration script
-RUN bash /usr/local/bin/image-config.sh
-
-USER wso2user
-WORKDIR /mnt
-
-# Expose transport ports
+# defined arguments and defaults
+ARG USER=wso2user
+ARG USER_ID=1000000000
+
+ARG FILES=./files
+ARG USER_HOME=/home/wso2user
+ARG WSO2_SERVER=wso2is
+ARG WSO2_SERVER_VERSION=5.3.0
+ARG WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}.zip
+ARG JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
+ARG JAVA_HOME=${USER_HOME}/java
+
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends --no-install-suggests \
+       	   zip \
+           unzip \
+           telnet \
+           iproute2 \
+	   sudo && rm -rf /var/lib/apt/lists/*
```

Comment:
The pull request looks good Shariq! I think it would be better to align above lines with the same indentation.